scrapy/contrib/loader/processor.py

```diff
@@ -63,6 +63,31 @@ def __call__(self, values):
         return values
 
 
+class JsonProcessor(object):
+    """
+        Given a jmespath, will search for it in the given scraped value
+        Requires : jmespath, ast
+    """
+    def __init__(self, jpath):
+        self.json_path = jpath
+        try:
+            import jmespath
+            import ast
+            self.astModule = ast
```

Comment:
Just save the function you are going to use.  `self.ast_literal_eval = ast.literal_eval`.
Also fix naming, we use camel case just for classes.